src/test/resources/discovery/test-manifest-sample-1.ttl

```diff
@@ -1,76 +1,54 @@
-    dcterms:title "Group 1"@en ;
-    dcterms:description "Description 1"@en ;
-    td:specificationReference <https://example.org/specification1#spec1> ;
-    dcterms:hasPart
-        <https://example.org/dummy/group1/feature1>,
-        <https://example.org/dummy/group1/feature2>,
-        <https://example.org/dummy/group1/feature3>
-.
-
-<https://example.org/dummy/group1/feature1>
+manifest:group1-feature1
     a td:TestCase ;
-    dcterms:subject "MUST" ;
+    spec:requirementReference <https://example.org/specification1#spec1> ;
     td:reviewStatus td:accepted ;
-    dcterms:title "Group 1 Test case 1" .
+    dcterms:title "Group 1 Test case 1" ;
+    spec:testScript <https://example.org/dummy/group1/feature1> .
 
-<https://example.org/dummy/group1/feature2>
+manifest:group1-feature2
     a td:TestCase ;
-    dcterms:subject "MUST" ;
+    spec:requirementReference <https://example.org/specification1#spec1> ;
     td:reviewStatus td:accepted ;
-    dcterms:title "Group 1 Test case 2" .
+    dcterms:title "Group 1 Test case 2" ;
+    spec:testScript <https://example.org/dummy/group1/feature2> .
 
-<https://example.org/dummy/group1/feature3>
+manifest:group1-feature3
     a td:TestCase ;
-    dcterms:subject "MAY" ;
+    spec:requirementReference <https://example.org/specification1#spec1> ;
     td:reviewStatus td:unreviewed ;
-    dcterms:title "Group 1 Test case 3" .
-
-manifest:group2
-    a td:SpecificationTestCase ;
-    dcterms:title "Group 2"@en ;
-    dcterms:description "Description 2"@en ;
-    td:specificationReference <https://example.org/specification1#spec2> ;
-    td:preCondition "sf1" ;
-    dcterms:hasPart
-        <https://example.org/dummy/group2/feature1>
-.
+    dcterms:title "Group 1 Test case 3" ;
+    spec:testScript <https://example.org/dummy/group1/feature3> .
 
-<https://example.org/dummy/group2/feature1>
+manifest:group2-feature1
     a td:TestCase ;
-    dcterms:subject "MUST" ;
+    spec:requirementReference <https://example.org/specification1#spec2> ;
     td:reviewStatus td:accepted ;
-    dcterms:title "Group 2 Test case 1" .
-
-manifest:group3
-    a td:SpecificationTestCase ;
-    dcterms:title "Group 3"@en ;
-    dcterms:description "Description 3"@en ;
-    td:specificationReference <https://example.org/specification1#spec3> ;
-    td:preCondition "sf1", "sf2" ;
-    dcterms:hasPart
-        <https://example.org/dummy/group3/feature1>,
-        <https://example.org/dummy/group3/feature2>
-.
+    dcterms:title "Group 2 Test case 1" ;
+    td:preCondition "sf1" ;
+    spec:testScript <https://example.org/dummy/group2/feature1> .
 
-<https://example.org/dummy/group3/feature1>
+manifest:group3-feature1
     a td:TestCase ;
-    dcterms:subject "MUST" ;
+    spec:requirementReference <https://example.org/specification1#spec3> ;
     td:reviewStatus td:accepted ;
-    dcterms:title "Group 3 Test case 1" .
+    dcterms:title "Group 3 Test case 1" ;
+    td:preCondition "sf1", "sf2" ;
```

Comment:
Can you give clear precondition as example to get a better feel for its purpose here?